pkg/cmd/util/output/backup_describer.go

```diff
@@ -163,6 +164,18 @@ func DescribeBackupStatus(d *Describer, status v1.BackupStatus) {
 
 	d.Println()
 	d.Printf("Expiration:\t%s\n", status.Expiration.Time)
+	d.Println()
+	// "<never>" output should only be applicable for backups that failed validation
+	if status.StartTimestamp.Time.Equal(time.Time{}) {
```

Comment:
`status.StartTimestamp.Time.IsZero()`

Reply:
Thanks. btw only the latest commit in this PR is new - the first one is from #564.